Source/Editor/Project/Project.cpp

```diff
@@ -498,13 +500,22 @@ void Project::InitializeDefaultProject()
 
 void Project::InitializeResourceCache()
 {
-    auto cache = GetSubsystem<ResourceCache>();
+    const auto engine = GetSubsystem<Engine>();
+    const auto cache = GetSubsystem<ResourceCache>();
     cache->ReleaseAllResources(true);
     cache->RemoveAllResourceDirs();
     cache->AddResourceDir(dataPath_);
     cache->AddResourceDir(coreDataPath_);
     cache->AddResourceDir(cachePath_);
     cache->AddResourceDir(oldCacheState_.GetEditorData());
+
+    const auto vfs = GetSubsystem<VirtualFileSystem>();
+    vfs->UnmountAll();
+    vfs->MountDir(oldCacheState_.GetEditorData());
```

Comment:
I vote for MountDirectory shortcut as well

Reply:
But "Dir" is very popular word in the codebase...
GetAppPreferencesDir
          SetCurrentDir
          CreateDir
          GetCurrentDir
          DirExists
          ScanDir
          ScanDirAdd
          GetProgramDir
          GetUserDocumentsDir
          GetAppPreferencesDir
          CopyDir
Why this would be different?

Reply:
Indeed, it should remain consistent.